### PR TITLE
#158: implement calculate average stars and forks for repos

### PR DESCRIPTION
**Pull request by Yegorov**

### CODE REVIEW

File: judges/quality-of-service/simple-collect.yml [hunk 1]
```diff
@@ -39,4 +39,7 @@ expected:
   - /fb/f[average_issue_lifetime = 0]
   - /fb/f[average_pull_lifetime = 0]
   - /fb/f[average_release_interval != 0]
-  - /fb/f[average_build_success_rate != 0]
\ No newline at end of file
+  - /fb/f[average_build_success_rate != 0]
+  - /fb/f[average_build_success_rate != 0]
```
Comment (yegor256) [inline judges/quality-of-service/simple-collect.yml:43]: @Yegorov two times the same line. typo?

**Review by yegor256 — CHANGES_REQUESTED** [review]
looks like a typo to me

File: judges/quality-of-service/simple-collect.yml (hunk 1 above)
Comment (Yegorov) [reply judges/quality-of-service/simple-collect.yml]: @yegor256 Yes, a typo, did not notice

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@Yegorov thanks!

**Comment by 0crat** [comment]
@Yegorov Thanks for the review! You've earned +5 points for this: +25 as a basis; -40 for reviewing your own contribution; +20 to give you at least something.

**Comment by 0crat** [comment]
@Yegorov Thanks for the contribution! You've earned +20 points. Please, [keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) them coming. Your running balance is [+5](https://zerocracy.github.io/judges-action/zerocracy-vitals.html).

**Comment by 0crat** [comment]
@yegor256 Hello! Thank you for your code review. As per our team policy, you've earned +15 points: +25 base points, with a -10 deduction due to the low number of comments (only 2). While we appreciate your effort, more detailed feedback would be valuable for the team. Your current balance stands at [-581](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the good work, and remember that providing thorough comments can boost your points in future reviews!

**Pull request merged**